go/go.sum

```diff
@@ -619,6 +619,10 @@ github.com/camunda/zeebe/clients/go/v8 v8.2.7 h1:fkL1ExWIqsVMg56nGDl/XfWf7W4PE3l
 github.com/camunda/zeebe/clients/go/v8 v8.2.7/go.mod h1:epiKnbOo1QeAD7BhdQoDNsdE0dJ2qoAx4uGxob1gHAQ=
 github.com/camunda/zeebe/clients/go/v8 v8.2.8-0.20230703113212-3628a8f1f24e h1:oYqV4Bhp7BoM1R+/dv5+xwXrnCPoxxfQ3z4bfFXckUk=
 github.com/camunda/zeebe/clients/go/v8 v8.2.8-0.20230703113212-3628a8f1f24e/go.mod h1:epiKnbOo1QeAD7BhdQoDNsdE0dJ2qoAx4uGxob1gHAQ=
+github.com/camunda/zeebe/clients/go/v8 v8.2.9 h1:LhfAslXwMxDlUY4eZRyZ42azp6g9cWoMM9BKh8dBId0=
```

Comment:
This looks a bit weird, this should be v8.2.10 too, right? @megglos

Reply:
the 8.2.10 entries as re in line 624/625 I was also first confused but it seems this keeps track of previous versions? (I updated a branch where 8.2.9 was already set)

Reply:
Oh I think a `go mod tidy` fixes this, at least for me.

Reply:
will do!